components/eamxx/src/share/util/scream_setup_random_test.hpp

```diff
@@ -18,7 +18,7 @@ inline int get_random_test_seed(const ekat::Comm* comm=nullptr)
 
   std::random_device rdev;
   const int catchRngSeed = Catch::rngSeed();
-  int seed = catchRngSeed==0 ? rdev() : catchRngSeed;
+  int seed = catchRngSeed==0 ? rdev()/2 : catchRngSeed;
```

Comment:
This change is b/c you can't do `./my_test --rng-seed -123`, since the cmd line arg parser interprets `-123` as a flag, not as the value for `--rng-seed`. Since `rdev()` is an unsigned int, assigning it to an int can overflow. But if we divide by 2, we are SURE it will be <=2^31-1, which is in the range of int, guaranteeing that seed>=0.